src/ATE_sammy/ate_sammy/templates/base_common_template.jinja2

```diff
@@ -115,9 +115,12 @@ class Context:
 def apply_configuration(feature_dict):
     for name, instance in feature_dict.items():
         config_file_path = pathlib.Path(os.path.join(pathlib.Path(__file__).parent.absolute(), "..", f"{name}.json"))
```

Comment:
the jinja2-template contains a mixture, i.e. it imports os and pathlib at the same time. It would be nice if you could remove import os in favour of pathlib and adapt the code accordingly.